src/data/schedules.json

```diff
@@ -3,7 +3,8 @@
 		"name": "Patriot Hybrid",
 		"isSpecial": false,
 		"dates": [
-			"weekdays"
+			"1/27/2021",
+			"Weekdays & !1/26/2021"
```

Comment:
I think the `1/27/2020` isn't necessary because it should be included in the `weekdays & !1/26/2021`. Also ideally, the `& !1/26/2021` shouldn't be necessary either since the schedule `Remote Learning` specifies `1/26/2021` and that should override this schedule. Let me know if you observed it behaving differently though and I'll see if I can fix it :)

Reply:
I can confirm that it works with those removed lines. That was just extra code I should have removed. Thanks

Reply:
Perfect, thanks!